vllm/engine/multiprocessing/client.py

```diff
@@ -503,7 +509,8 @@ async def _process_request(
         request_id: str,
         lora_request: Optional[LoRARequest] = None,
         trace_headers: Optional[Mapping[str, str]] = None,
-        prompt_adapter_request: Optional[PromptAdapterRequest] = None
+        prompt_adapter_request: Optional[PromptAdapterRequest] = None,
+        priority: int = 0,
```

Comment:
Did you forget to use this in `serving_embedding.py`?

Reply:
Yes, apparently. Completely forgot about the embedding endpoints. Will fix.

Reply:
Priority was also missing from `encode` in the client-protocol and from the `AsyncLLMEngine`. Fixed this and edited the PR description accordingly.